docs/content/users/configuration/hooks.md

```diff
@@ -144,12 +150,12 @@ hooks:
 ```yaml
 hooks:
   pre-start:
-    # Install composer dependencies using composer on host system
+    # Install Composer dependencies using composer on host system
     - exec-host: "composer install"
```

Comment:
Rather than get rid of the `composer install` step entirely, I moved it into the `post-start` hook, which I hope is more appropriate:

```yaml
hooks:
  post-start:
    # Install Composer dependencies from the web container
    - composer: install
```